src/components/RichText.ts

```diff
@@ -175,7 +175,7 @@ export class RichText extends Component<RichTextProps> {
         if (this.quill) {
             const value = this.quill.root.innerHTML !== this.undoDefault ? this.quill.root.innerHTML : "";
             if (this.props.value !== value) {
-                this.props.onChange(value);
+                this.props.onChange(value, event as KeyboardEvent);
```

Comment:
Unless you're sure this works in every browser, do it different. In fact, even if it works in all browsers, do it different if possible. Otherwise, add a comment about what it is and where it's coming from. Basically, prefix it with `window` ... `window.event`. A check would be nice too since it's not always available.

Reply:
Done